aten/src/ATen/cudnn/Descriptors.h

```diff
@@ -339,7 +339,7 @@ union Constant
   double d;
   Constant(cudnnDataType_t dataType, double value) {
     if (dataType == CUDNN_DATA_HALF || dataType == CUDNN_DATA_FLOAT) {
-      f = (float) value;
+      f = static_cast<float>(value);
```

Comment:
Curious, why is the static_cast necessary here?

Reply:
Simply better to use the explicit C++ syntax over C-style cast, I believe.